torch/types.py

```diff
@@ -62,6 +62,9 @@
 
 # Meta-type for "numeric" things; matches our docs
 Number: TypeAlias = Union[int, float, bool]
+# tuple for isinstance(x, Number) checks.
+# FIXME: refactor once python 3.9 support is dropped.
+_Number = (int, float, bool)
```

Comment:
What's the feature we can use to drop this after 3.9? Just isinstance checking of Unions?

Reply:
Yes, as per https://peps.python.org/pep-0604/ you can just replace `_Number` with `Number` itself then.